src/app/customer/customer-service.service.ts

```diff
@@ -0,0 +1,9 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
```

Comment:
dont user root providation 
```suggestion
@Injectable()
```

Reply:
this is because i want to use it in the root level